packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -240,8 +241,11 @@ class Checkbox extends StatefulWidget {
   /// {@macro flutter.widgets.Focus.autofocus}
   final bool autofocus;
 
-  /// The Border Radius of the checkbox.
-  final double borderRadius;
+  /// The Border of the checkbox.
+  final RoundedRectangleBorder? shape;
+
+  /// The TextDirection of the checkbox.
```

Comment:
Please don't just reiterate the property name as documentation. Explain how this parameter affects the checkbox's visuals.

Please explain what this property's default value is in terms of this parameter, and the theme.